syz-manager/html.go

```diff
@@ -241,7 +254,7 @@ func (mgr *Manager) httpCoverCover(w http.ResponseWriter, r *http.Request,
 			}
 			progs = append(progs, cover.Prog{
 				Data: string(inp.Prog),
-				PCs:  coverToPCs(rg, inp.Cover),
+				PCs:  coverToPCs(rg, ifFilter(inp.Cover)),
```

Comment:
I think it will be more consistent/better to filter it in the case above as well (when we have "input").